RELEASE_NOTE.md

```diff
@@ -1,5 +1,25 @@
 # OpenPAI Release Note
 
+## Nov 2020 (version 1.3.0)
+
+- Marketplace
+  - New templates in marketplace (https://github.com/microsoft/openpaimarketplace/issues/60)
+- HiveD Scheduler
+  - Support cluster autoscale with HiveD scheduler on AKS (#4868)
+  - Support dynamic sku types for different vc on webportal (#4900)
+- Advanced job debug mode
+  - Add per task retry history (https://github.com/microsoft/frameworkcontroller/pull/62, #4958, #4966)
```

Comment:
These issue link are not showed as hyperlink.

Reply:
not sure whether it's a regression on markdown rendering, as previous release notes all used this format

Reply:
this is not a breaking issue for the release, I will send a docs fix in another PR